packages/components/src/Modal/src/CustomModal.tsx

```diff
@@ -29,6 +27,10 @@ export interface CustomModalProps extends
      * @default "md"
      */
     size?: ResponsiveProp<"sm" | "md" | "lg" | "xl" | "fullscreen" | "fullscreenTakeover">;
+    /**
+     * The props of the overlay
+     */
+    overlayProps?: ModalOverlayProps;
```

Comment:
usually it's Partial<ModalOverlayProps> since if there is a required prop, we already provided it by default